test/queue-test.js

```diff
@@ -572,6 +572,126 @@ describe('Queue', (it) => {
     });
   });
 
+  it.describe('getJobs', (it) => {
+    it('gets waiting jobs', async (t) => {
+      const queue = t.context.makeQueue();
+
+      const insertedJobs = [];
+      for (let i = 0; i < 3; i++) {
+        let job = await queue.createJob({foo: 'bar'}).save();
+        insertedJobs.push(job.toData());
+      }
+
+      const jobs = await queue.getJobs('waiting', {start: 0, end: 2});
+      for (let job of jobs) {
+        t.true(insertedJobs.includes(job.toData()));
+      }
+    });
+
+    it('gets active jobs', async (t) => {
+      const queue = t.context.makeQueue();
+
+      await queue.createJob({foo: 'bar'}).save();
+
+      queue.process(async (job) => {
+        const jobs = await queue.getJobs('active', {start: 0, end: 100});
+        t.is(jobs[0].toData(), job.toData());
+      });
+
+      await helpers.waitOn(queue, 'succeeded');
+    });
+
+    it('gets delayed jobs', async (t) => {
+      const queue = t.context.makeQueue();
+
+      queue.process(async () => {});
+
+      const job = await queue.createJob({foo: 'bar'})
+        .delayUntil(Date.now() + 1000)
```

Comment:
Increase this to like `100000` - doesn't hurt, and these tests are pretty intensive